modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/CheckboxMultiSelect.tsx

```diff
@@ -76,6 +76,7 @@ function CheckboxMultiSelect({
 					<div className="autofit-row autofit-row-center">
 						<div className="autofit-col mr-3">
 							<ClayCheckbox
+								label={item.name}
```

Comment:
Hello @markocikos ,thanks for checking the code, for the source formatting issue I just ran the `gradle formatSource` and it fixed that issue.
As for the Label, it doesn't actually add a second label because the first one isn't taken into consideration when I scan the page using axe DevTools, so I added the label and it works.
I will push again because I changed the label with aria-label instead but the reasoning is the same.